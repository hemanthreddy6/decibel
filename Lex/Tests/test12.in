audio track <- load "track.wav";
audio modifiedTrack <- (track >> 2) * 1.2 & DISTORT(3); 
/*
The above expression speeds up the track ny a factor of 2, scales all the amplitudes
by a factor of 1.2 and the distorts the track based on some distortion function with
3 as an input.
*/
play modifiedTrack;
