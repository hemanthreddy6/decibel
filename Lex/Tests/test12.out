AUDIO
IDENTIFIER
<-
LOAD
STRING
;
AUDIO
IDENTIFIER
<-
(
IDENTIFIER
>>
NUMBER
)
*
FLOAT
&
IDENTIFIER
(
NUMBER
)
;
PLAY
IDENTIFIER
;
